backend/src/interfaces/IPendingCoursesList/IPendingCourse.ts

```diff
@@ -0,0 +1,11 @@
+import { ObjectId } from 'mongoose';
+
+interface IPendingCourse {
+  _id?: string;
+  user: ObjectId;
+  course: ObjectId;
+  time: Date;
+  status: ObjectId;
+}
+
```

Comment:
pending course it's just omited type of clientCourses
 you can inherit this type from clientCourse